OutOfSchool/OutOfSchool.DataAccess/Repository/WorkshopRepository.cs

```diff
@@ -11,9 +8,10 @@ namespace OutOfSchool.Services.Repository
     public class WorkshopRepository : EntityRepository<Workshop>, IWorkshopRepository
     {
         private readonly OutOfSchoolDbContext db;
```

Comment:
(To write less code and make linter happier) 
You could use getter-only property instead of backing field and expression-bodied getter-only property, like in my example (`TestPropertySetterOutsideCtor()` is used only to show that setting of the property is allowed only via ctor, it shouldn't be included in your code):

![image](https://user-images.githubusercontent.com/85107137/134168688-a7835d30-1004-4b75-abbd-422e5fa2ad83.png)

Reply:
And now I've realized, that you are using `OutOfSchoolDbContext db` everywhere. Then at least add empty line between field and property for linter.